server.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * This file is here only to show you how to proceed if you would
+ * like to turn your application into a standalone executable.
+ */
+
+import Fastify from 'fastify'
+
+// Import library to exit fastify process, gracefully (if possible)
+import closeWithGrace from 'close-with-grace'
+
+// Import your application as a normal plugin.
+import appService from './app.js'
+
+// Instantiate Fastify with some config
+const app = Fastify({
+  logger: true
+})
+
+async function init () {
+  // Register your application as a normal plugin.
+  app.register(appService)
+
+  // delay is the number of milliseconds for the graceful close to finish
+  const closeListeners = closeWithGrace({ delay: process.env.FASTIFY_CLOSE_GRACE_DELAY || 500 }, async function ({ signal, err, manual }) {
+    if (err) {
+      app.log.error(err)
+    }
+    await app.close()
+  })
+
+  app.addHook('onClose', (instance, done) => {
+    closeListeners.uninstall()
```

Comment:
This isn't mentioned on the readme here: https://github.com/mcollina/close-with-grace. I think you are correct with this uninstall I believe you. Unless Matteo is saying this is no longer needed. Anyhow, maybe we should also create a PR for `mcollina/close-with-grace` to update the README.

Reply:
Maybe this example need an update so:
https://github.com/fastify/fastify-cli?tab=readme-ov-file#migrating-out-of-fastify-cli-start

Reply:
@mcollina

Reply:
@mcollina

Reply:
PR open [here](https://github.com/mcollina/close-with-grace/pull/14), I resolve the conversation for now.

Reply:
Matteo is saying it's not needed, which is strange. Meaning the documentation of fastify-cli is incorrect?...